dispatchrouter/qpid/qdrouterd-with-broker.conf

```diff
@@ -0,0 +1,147 @@
+router {
```

Comment:
this file seem to be the same as the JSON based one, isn't it?
Is this required?